recipe/meta.yaml

```diff
@@ -19,7 +19,7 @@ requirements:
     - gcc
     - libnetcdf 4.4.*
     - proj4 4.9.3
-    - gdal 2.2.*
+    - libgdal 2.2.*
```

Comment:
This will prevent a simple `ncl` installation from downloading `numpy` and other python stuff from the `gdal` python wrapper. My guess is that `ncl` does not really need that.